source/features/fit-textareas.tsx

```diff
@@ -10,6 +10,10 @@ function inputListener(event: Event): void {
 }
 
 function watchTextarea(textarea: HTMLTextAreaElement): void {
+	if (!textarea) {
+		return;
+	}
+
```

Comment:
This should never be possible. This function has to be called with `textarea`. 

Only `fitPrCommitMessageBox` calls it with a possibly-non-existing element (via `select`) and even then it shouldn’t: how can the commit message field be missing from the merge box?

My guess is that onPrMergePanelOpen is firing at the wrong time. *That* needs to be fixed. 

In fact the other function has the same exact problem and the same exact solution.

Reply:
>My guess is that onPrMergePanelOpen is firing at the wrong time. That needs to be fixed.

Yes that is the issue.

Any idea's i should look into?

Reply:
https://github.com/sindresorhus/refined-github/blob/7f1f2c52d8deeb0fddb7a7322af7dd090ac19461/source/libs/on-pr-merge-panel-open.ts#L12

This was supposed to do that